src/js/tracing/ops.ts

```diff
@@ -4,6 +4,7 @@ export const NAVIGATION = 'navigation';
 
 export const UI_LOAD = 'ui.load';
 export const UI_ACTION_TOUCH = 'ui.action.touch';
+export const UI_ACTION_GESTURE = 'ui.action.gesture';
```

Comment:
Can we build it dynamically? eg `ui.action.*` where * is the gesture name.

Reply:
Yes.